aws/resource_aws_glue_catalog_table.go

```diff
@@ -16,6 +16,7 @@ func resourceAwsGlueCatalogTable() *schema.Resource {
 		Read:   resourceAwsGlueCatalogTableRead,
 		Update: resourceAwsGlueCatalogTableUpdate,
 		Delete: resourceAwsGlueCatalogTableDelete,
+		Exists: resourceAwsGlueCatalogTableExists,
```

Comment:
Sorry this might not be clear since we have a few resources that implement it, but we are preferring to skip implementing the `Exists` function in this provider. Could you remove this? It usually introduces unnecessary logic duplication as the beginning of each resource `Read` function should handle the "existence" testing logic and triggering recreation via the `d.SetId("")` and `return nil` pattern. Thanks!